src/cmd/mod.rs

```diff
@@ -153,22 +153,31 @@ fn collect_addresses(files: Vec<PathBuf>, mut addresses: Vec<PublicKey>) -> Resu
 }
 
 fn get_seed_words(seed_type: &mnemonic::SeedType) -> Result<Vec<String>> {
-    use dialoguer::Input;
-    let split_str = |s: &String| s.split_whitespace().map(|w| w.to_string()).collect();
-    let word_string = Input::<String>::new()
-        .with_prompt("Space separated seed words")
-        .validate_with(|v: &String| {
-            let word_list = split_str(v);
-            match mnemonic::mnemonic_to_entropy(word_list, seed_type) {
-                Ok(_) => Ok(()),
-                Err(err) => Err(err),
-            }
-        })
-        .interact()?;
-    Ok(word_string
-        .split_whitespace()
-        .map(|w| w.to_string())
-        .collect())
+    match env::var("HELIUM_WALLET_SEED_WORDS") {
+        Ok(val) => {
+            let split_str = |s: &String| s.split_whitespace().map(|w| w.to_string()).collect();
+            let word_string = split_str(&val);
+            Ok(word_string)
+        },
+        _ => {
+            use dialoguer::Input;
+            let split_str = |s: &String| s.split_whitespace().map(|w| w.to_string()).collect();
+            let word_string = Input::<String>::new()
+                .with_prompt("Space separated seed words")
+                .validate_with(|v: &String| {
+                    let word_list = split_str(v);
+                    match mnemonic::mnemonic_to_entropy(word_list, seed_type) {
+                        Ok(_) => Ok(()),
+                        Err(err) => Err(err),
+                    }
+                })
+                .interact()?;
+            Ok(word_string
```

Comment:
use the common split_str function here too if you could, please

Reply:
and done :)

Reply:
you didn't actually do it here..

Reply:
Why not? Please look at the diff, because I forced pushed to my branch and this view might be outdated.

Reply:
Done. Sorry for the misunderstanding.